server-common/src/main/java/org/apache/gravitino/server/authorization/jcasbin/JcasbinAuthorizer.java

```diff
@@ -309,8 +307,7 @@ private boolean loadPrivilegeAndAuthorize(
         userId = userEntity.id();
         metadataId = MetadataIdConverter.getID(metadataObject, metalake);
       } catch (Exception e) {
-        LOG.debug("Can not get entity id", e);
-        return false;
+        throw new RuntimeException(e);
```

Comment:
If a certain metadata does not exist in Gravitino, an exception will be thrown here. In fact, it should return `false`, because throwing an exception would interrupt the list metadata operation.

Reply:
As for `loadPrivilegeAndAuthorize`, @hdygxsj  could you provide the exact meaning of return false?

Reply:
We might want to swallow some exceptions and return false, while letting the remaining exceptions propagate. 

Such as,

```
java.lang.RuntimeException: org.apache.gravitino.exceptions.NoSuchEntityException: No such schema entity: default
```